include/seqan3/range/view/translation.hpp

```diff
@@ -33,256 +33,567 @@
+ * \tparam urng_t The type of the range being processed.
+ * \param urange The range being processed.
+ * \param tf A value of seqan3::tanslation_frames that indicates the desired frames.
+ * \returns A range containing frames with aminoacid sequence. See below for the properties of the returned range.
  * \ingroup view
  *
  * \details
  *
- * This view can be used to translate nucleotide sequences into single forward aminoacid sequences.
-
- * \par View properties
+ * This view can be used to translate nucleotide sequences into aminoacid sequences:
+ * translation_frames::FIRST_FRAME_FORWARD       - First forward frame starting at position 0
+ * translation_frames::FIRST_FRAME_REVERSE       - First reverse frame starting at position 0
+ * translation_frames::SECOND_FRAME_FORWARD      - Second forward frame starting at position 1
+ * translation_frames::SECOND_FRAME_REVERSE      - Second reverse frame starting at position 1
+ * translation_frames::THIRD_FRAME_FORWARD       - Third forward frame starting at position 2
+ * translation_frames::THIRD_FRAME_REVERSE       - Third reverse frame starting at position 2
+ *
+ * ### View properties
  *
- * |                     | `irng_t` (range input type)           | `rrng_t` (range return type)                              |
- * |---------------------|---------------------------------------|-----------------------------------------------------------|
- * | range               | seqan3::input_range_concept           | seqan3::view_concept + all range concepts met by `irng_t` |
- * | `range_reference_t` | seqan3::nucleotide_concept            |                                                           |
+ * |                     | `urng_t` (underlying range type)                                   | `rrng_t` (range return type)                                |
+ * |---------------------|--------------------------------------------------------------------|-------------------------------------------------------------|
+ * | range               | seqan3::sized_range_concept && seqan3::random_access_range_concept | seqan3::view_concept && seqan3::random_access_range_concept |
+ * | `range_reference_t` | seqan3::nucleotide_concept                                         | seqan3::aa27                                                |          |
  *
- * * `irng_t` is the type of the range modified by this view (input).
+ * * `urng_t` is the type of the range modified by this view (input).
  * * `rrng_type` is the type of the range returned by this view.
  * * for more details, see \ref view.
  *
- * \par Example
+ * ### Example
  *
  * Operating on a range of seqan3::dna5:
  * ```cpp
  * dna5_vector vec{"ACGTACGTACGTA"_dna5};
  *
- * // single frame translation
- * auto v1 = vec | view::translate_single;                     // == [T,Y,V,R]
+ * // First forward frame
+ * auto v1 = vec | view::translate_single(translation_frames::FIRST_FRAME_FORWARD);                                  // == [T,Y,V,R]
```

Comment:
I don't think these are the current enum labels anymore, don't they begin with `FWD..`?